atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/sweep/AbstractSweeperTest.java

```diff
@@ -341,9 +344,8 @@ public void testSweepResultsIncludeMinimumTimestamp() {
     }
 
     @Test
-    @Ignore
     /*
-     * These tests have been ignored pending internal ticket 94009
+     * These tests were ignored pending internal ticket 94009
```

Comment:
Does this PR resolve the internal ticket? 

I'm in favor of deleting this whole comment block.